src/store/helpSlice.ts

```diff
@@ -0,0 +1,23 @@
+import { createSlice } from '@reduxjs/toolkit';
+import { INITIAL_HELP } from './initialStates';
+
+export type HelpSlice = {
+  [key: string]: Help;
+};
+
+export type Help = {
```

Comment:
I think we need a place to store images associated with each troubleshooting page. This will likely be an array, as there can be multiple for each step!